eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md

```diff
@@ -136,15 +136,23 @@ source ./terraform/setup_env.sh
  . terraform\setup_env.ps1
 ```
 
+If you want to run the program in debug mode, you can save the output value. 
+
 ## Run Locally
 
+### Run the program directly
+
 In your terminal, run `mvn clean spring-boot:run`.
 
 
 ```shell
 mvn clean spring-boot:run
 ```
 
+### Run the program in debug mode
+
+You can debug your program by adding the saved output values or finding the required parameters in the `terraform.tfstate` file in the program's terraform directory and adding them to the tool's environment variables or the program's `application.yaml` file.
```

Comment:
suggest delete this
<img width="949" alt="image" src="https://user-images.githubusercontent.com/4465723/158092709-bb11bbad-3c6c-4f4e-8765-142e404eeed4.png">


and better add a pic to demonstrate how to do it.